docs/source/cli_help/status.rst

```diff
@@ -58,7 +58,7 @@ Using colors can be disabled with a ``--color`` flag set to ``never``.
 With ``--color=always``, git machete always emits colors and with ``--color=auto``, it emits colors only when standard output is connected to a terminal.
 ``--color=auto`` is the default. When colors are disabled, relation between branches is represented in the following way (not including the hash-comments):
 
-.. code-block::
+.. code-block:: git
```

Comment:
Does this really work? I'm pretty sure it doesn't exist as available lexers.

Reply:
Oh! see my comment & Andrzej's response, same surprise on my side ;)